src/App.js

```diff
@@ -198,14 +199,16 @@ export default class App extends Base {
           if (!this.appContinue) {
             const systemui = new URLSearchParams(window.location.search).get('systemui');
 
-            if (systemui) {
-              try {
+            try {
+              if (systemui) {
                 AcknowledgeChallenge.provide('xrn:firebolt:capability:usergrant:acknowledgechallenge', new AckChallengeProviderDelegater(this));
                 Keyboard.provide('xrn:firebolt:capability:input:keyboard', new KeyboardProviderDelegater(this));
                 PinChallenge.provide('xrn:firebolt:capability:usergrant:pinchallenge', new PinChallengeProviderDelegater(this));
-              } catch (err) {
-                logger.error('Could not set up providers' + err, 'LoadedState');
+              } else {
```

Comment:
As discussed in the thread, if we can create a function that can decide to provide/not provide support for the provider based on the flag coming in Parameter.initialization or onNavigateTo via intent

Reply:
Is that ok we can handle this point and 
2.Doesn't register as a provider (platform should respond with not available)
I believe we don't have this support for any provider as of now but will have to design it
3. Registers as a provider but just never responds (platform should timeout the provider and respond with not available)
In new ticket https://ccp.sys.comcast.net/browse/FIRECERT-2148?